website/pages/docs/drivers/exec.mdx

```diff
@@ -93,6 +93,13 @@ If you are receiving the error:
 and using the exec driver, check to ensure that you are running Nomad as root.
 This also applies for running Nomad in -dev mode.
 
+## Plugin Options
+
+* `no_pivot_root` - Defaults to `false`. Changing this to `true` will pass the
+  `NoPivotRoot` configuration option to `libcontainer` which will fall back to
+  using the `msMoveRoot` function for isolation. This is useful for systems
+  where the root is on a ramdisk.
```

Comment:
Some of this feels like an implementation details (e.g. the bits about libcontainer).  How do you feel about something like:

```md
* `no_pivot_root` - Defaults to `false`. When `true`, the driver uses `chroot` for file system
  isolation without `pivot_root`.  This is useful for systems where the root is on a ramdisk.
```

Reply:
Sure, that sounds good! I just updated the PR.